doc/source/ray-core/fault_tolerance/tasks.rst

```diff
@@ -23,6 +23,22 @@ another task that depends on the object.
   :start-after: __task_exceptions_begin__
   :end-before: __task_exceptions_end__
 
+For developer ergonomics, Ray raises the exception as an instance of both `RayTaskError`
```

Comment:
Should we just change line 17-19 to be 

```
This wrapped exception will be thrown to any worker that tries
to get the result, either by calling ``ray.get`` or if the worker is executing
another task that depends on the object. If the user's exception type can be subclassed, the wrapped exception is subclass of both RayTaskError and user's exception type so user can try catch either of them. Otherwise, the wrapped exception is just RayTaskError and the actual user's exception type can be accessed via the `cause` field of RayTaskError
```

something like that?